src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash';
+
+import {
+  MAX_CONTEXT_SIZE,
+  MIN_CONTEXT_SIZE,
+  QUERY_PARAMETER_KEYS,
+} from './constants';
+
+
+export function QueryParameterActionsProvider() {
```

Comment:
Makes sense. I don't feel too strongly either way, it's probably fine to leave as is.